exporter.go

```diff
@@ -267,6 +254,26 @@ func (e *callbackContainer) callback(ctx context.Context, msg realtime.Message)
 	return nil
 }
 
+// NewCollector creates a new Collector for the specified client and monitor
+func NewCollector(ctx context.Context, client *sense.Client, monitor int) *Collector {
```

Comment:
Do we need both of these variations on NewCollector?  Maybe just put timeout in NewCollector and just use one?